tests/helper.ts

```diff
@@ -9,15 +9,46 @@
  **********************************************************************/
 
 import * as path from 'path';
-import * as vscode from '@theia/plugin';
+import * as theia from '@theia/plugin';
 
 export const extensionID = 'redhat.java';
 
 export function closeAllOpenFiles() {
-    vscode.window.visibleTextEditors.map(file => vscode.commands.executeCommand('workbench.action.closeActiveEditor', file.document.uri));
+    theia.window.visibleTextEditors.map(file => theia.commands.executeCommand('workbench.action.closeActiveEditor', file.document.uri));
 }
-export async function sleep(ms: number) {
-    return new Promise(resolve => setTimeout(resolve, ms));
+export const getSrcDocPath = (p: string) => path.resolve('/projects/Che-Java-Tests/testWorkspace/src/main/java/org/my/sample', p);
+export const getSrcDocUri = (p: string) => theia.Uri.file(getSrcDocPath(p));
+
+// Try to restore Position object from its serialized content
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function revivePosition(position: any): theia.Position {
```

Comment:
should these revive functions available from theia namespace as I suspect you would have to copy it on every tests ?

Reply:
I've changed tests, now they don't use revive* functions